lib/common/xml.c

```diff
@@ -1746,11 +1752,14 @@ pcmk__xml_update(xmlNode *parent, xmlNode *target, xmlNode *update,
     CRM_CHECK(pcmk__xe_is(target, (const char *) update->name), return);
 
     if (as_diff == FALSE) {
-        /* So that expand_plus_plus() gets called */
+        // So that pcmk__xa_set_expand() gets called
         pcmk__xe_copy_attrs(target, update);
 
     } else {
-        /* No need for expand_plus_plus(), just raw speed */
+        /* No need for pcmk__xa_set_expand(), so use faster method
+         *
+         * @TODO pcmk__xe_copy_attrs() also checks ACLs. Should we do that here?
```

Comment:
> Probably even less worth it, but if the extra copy_attrs argument were a flag group, it could cover this case too

I'll ignore it since this block will go away when `apply_xml_diff()` goes away (hopefully in 3.0.0, as it's been deprecated since 2.0.5).